### PR TITLE
Added Stock functionality

### PR DESCRIPTION
**Pull request by lellebebbl**

### CODE REVIEW

File: Entity/StockInfo.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="cm_stock_info")
+ */
+class StockInfo extends CollmexObject
+{
+    /**
+     * @ORM\Column(type="string")
+     */
+    protected $type_identifier ='CMXPRD';
```
Comment (jdambacher) [inline Entity/StockInfo.php]: Please adopt the type identifier

File: Entity/StockInfo.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    protected $volume_unit;
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     */
+    protected $reorder_time;
+
+
+
+    public function getExtraInfo()
```
Comment (jdambacher) [inline Entity/StockInfo.php]: Is getExtraInfo() and parseRemark() required here? I think it's only used by product entities.

File: Services/CollmexManager.php [hunk 3]
```diff
@@ -113,7 +113,7 @@ public function prepareData($data)
             foreach ($data->getData() as $field) {
                 $strCSV .= $field . ";";
             }
-       }
+        }
```
Comment (jdambacher) [inline Services/CollmexManager.php]: Is the indent correct?

File: Services/CollmexManager.php [hunk 4]
```diff
@@ -167,6 +167,21 @@ public function getProducts($prodGroup = null, $priceGroup = null, $onlyModified
         return $response->getObjects();
     }
 
+    public function getStockInfo($onlyModified = 0, $companyId = 1, $systemName = 'Shop')
```
Comment (jdambacher) [inline Services/CollmexManager.php]: We should make this changeable by the developer user in the parameters.yml file. Please use in the meantime 'CoffeeBikeCollmexBundle' as systemName.

**Comment by lellebebbl** [comment]
@jdambacher Please, check again. Issues should be resolved

File: Services/CollmexManager.php (hunk 3 above)
Comment (lellebebbl) [inline Services/CollmexManager.php]: yes

File: Entity/StockInfo.php [hunk 5]
```diff
@@ -39,13 +39,7 @@ class StockInfo extends CollmexObject
...
      */
     protected $reorder_time;
-
-
-
-    public function getExtraInfo()
-    {
-        return $this->parseRemark();
-    }
+
```
Comment (jdambacher) [inline Entity/StockInfo.php]: Remove the dummy lines

**Pull request merged**